scripts/setup-ubuntu.sh

```diff
@@ -139,8 +139,8 @@ function install_conda {
   fi
 
   mkdir -p conda && cd conda
-  wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-$ARCH.sh
-  ${SUDO} bash Miniconda3-latest-Linux-$ARCH.sh -b -p $MINICONDA_PATH
```

Comment:
Can you set `SUDO=" "`(space) in the environment instead of removing it? I vaguely remember needing this for non root users.

Reply:
Nevermind, I see the path can be a user path.